pkgs/development/web/nodejs/v16.nix

```diff
@@ -1,8 +1,20 @@
-{ callPackage, openssl, python3, fetchpatch, enableNpm ? true }:
+{ callPackage, lib, overrideCC, pkgs, buildPackages, openssl, python3, fetchpatch, enableNpm ? true }:
 
 let
+  # Clang 16+ cannot build Node v14 due to -Wenum-constexpr-conversion errors.
+  # Use an older version of clang with the current libc++ for compatibility (e.g., with icu).
+  ensureCompatibleCC = packages:
```

Comment:
Does the node infra have something like a `common.nix` where this would fit?

Reply:
It does, but I don’t want to put stuff in there that’s only needed until the deprecated versions are removed.